plugins-weekly.ci.jenkins.io.txt

```diff
@@ -41,7 +39,6 @@ docker-commons:439.va_3cb_0a_6a_fb_29
 durable-task:523.va_a_22cf15d5e0
 echarts-api:5.4.3-2
 extended-read-permission:53.v6499940139e5
-favorite:2.208.v91d65b_7792a_c
```

Comment:
Note that this plugin is a dependency of `customizable-header`:

```[2024-01-09T18:47:08.050Z] #23 40.15 customizable-header depends on: 
[2024-01-09T18:47:08.051Z] #23 40.15 caffeine-api 3.1.8-133.v17b_1ff2e0599
[2024-01-09T18:47:08.051Z] #23 40.15 ionicons-api 56.v1b_1c8c49374e
[2024-01-09T18:47:08.051Z] #23 40.15 antisamy-markup-formatter 162.v0e6ec0fcfcf6
[2024-01-09T18:47:08.051Z] #23 40.15 cloudbees-folder 6.858.v898218f3609d
[2024-01-09T18:47:08.051Z] #23 40.15 favorite 2.208.v91d65b_7792a_c
```

Reply:
Same like above